### ISSUE TITLE
Gen VI Pokémon not included.

### ISSUE THREAD

**Issue by penman** [nolanlawson/pokedex.org#1]
[It seems PokéAPI has the data for Gen VI Pokémon](http://pokeapi.co/api/v1/pokemon/650), but they aren't included on pokedex.org.

**Comment by penman** [comment]
(I will attempt to PR for this but I'm waiting for the database to build)

**Comment by nolanlawson** [comment]
Yup! I did this because the data from PokéAPI was not 100% complete. Also there is some missing data that I had to fill in manually by scraping Bulbapedia. (See `src/js/shared/data` for some of the stuff I had to manually include.)

That said, I would *love* to have the Gen VI Pokémon in there. It should be pretty easy to implement as well; we can either just 1) modify the live Cloudant database, changes get synced automatically, or 2) rebuild the pouchdb-load files, then rename the IndexedDB database stores so that the old versions get overwritten (probably the easiest option, TBH).

**Comment by nolanlawson** [comment]
So in the ideal case, whatever you do should probably involve a fix to `src/assets` as well as the build scripts (which pull from the `src/js/shared/data` in some cases). Lemme know if you need help! :)

**Comment by Bokoblin** [comment]
Is the data complete now for the 6th generation ?
